docs/en/guide/configurations.md

```diff
@@ -142,54 +145,237 @@ globalConfig:
 ```
 
-### Inherit from CSI Node (deprecated) {#inherit-from-csi-node}
+</details>
 
-:::tip
-Starting from v0.24, CSI Driver can customize mount pods and sidecar containers in the [ConfigMap](#configmap), legacy method introduced in this section is not recommended.
-:::
+All supported configuration items and PVC selectors are listed in the above snippet, but the behavior of each item is introduced in below sections, please read before use.
 
-Mount pod specs are mostly inherited from CSI Node, for example if you need to enable `hostNetwork` for mount pods, you have to instead add the config to CSI Node:
+### Custom mount image {#custom-image}
 
-```yaml title="values-mycluster.yaml"
-node:
-  hostNetwork: true
+#### Via Configmap
+
+Minimum required version is CSI Driver v0.24.0. Upon modification, application pods or mount pods need to be re-created for changes to take effect. If you decide to re-create mount pod, be sure to enable [automatic mount point recovery](./configurations.md#automatic-mount-point-recovery) in advance, to avoid permanent loss of mount point within the application pod.
+
+```yaml {2-4}
+globalConfig:
+  mountPodPatch:
+    - ceMountImage: juicedata/mount:ce-v1.2.0
+      eeMountImage: juicedata/mount:ee-5.1.0-053aa0b
+```
+
+If you need to use a customized image, or would like to find the latest available JuiceFS mount image, refer to [customize container image](./custom-image.md).
+
+### Environment variable {#custom-env}
+
+#### Via Configmap
+
+Minimum required version is CSI Driver v0.24.5. Upon modification, application pods need to be re-created for changes to take effect.
+
+```yaml {2-6}
+  mountPodPatch:
+    - env:
+      - name: DEMO_GREETING
+        value: "Hello from the environment"
+      - name: DEMO_FAREWELL
+        value: "Such a sweet sorrow"
 ```
 
-After the change, newly created mount pods will use hostNetwork.
+#### Via Secret
 
-As mentioned earlier, "most" specs are inherited from CSI-node, this leaves component specific content like labels, annotations, etc. These fields will not work through inheritance so we provide separate methods for customization, read the next section for more.
+```yaml {11}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: juicefs-secret
+  namespace: default
+  labels:
+    # Add this label to enable secret validation
+    juicefs.com/validate-secret: "true"
+type: Opaque
+stringData:
+  envs: '{"BASE_URL": "http://10.0.0.1:8080/static"}'
+```
+
+### Resource definition {#custom-resources}
+
+#### Via Configmap
 
-### Customize via annotations (deprecated) {#others}
+Minimum required version is CSI Driver v0.24.0. Upon modification, application pods or mount pods need to be re-created for changes to take effect. If you decide to re-create mount pod, be sure to enable [automatic mount point recovery](./configurations.md#automatic-mount-point-recovery) in advance, to avoid permanent loss of mount point within the application pod.
```

Comment:
```suggestion
The minimum required version is CSI Driver v0.24.0. Upon modification, application pods or Mount Pods need to be re-created for changes to take effect. If you decide to re-create Mount Pods, be sure to enable [automatic mount point recovery](./configurations.md#automatic-mount-point-recovery) in advance, to avoid permanent loss of mount point within the application pod.
```